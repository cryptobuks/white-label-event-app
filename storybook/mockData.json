{
  "id": "scheduleItem",
  "author":   {
    "id": 8,
    "name": "Cedric Devroey",
    "imageUrl": "https://picsum.photos/300/250/?random"
  },
  "title": "Bridging the Gap",
  "location": "Arena",
  "date": "Tue Mar 06 2018 16:22:31 GMT+0100 (CET)",
  "labelColor": "pink",
  "labelValue": "Label"
}
