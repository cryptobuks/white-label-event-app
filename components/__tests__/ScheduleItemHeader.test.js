import React from 'react';
import renderer from 'react-test-renderer';
import { ScheduleItemHeader } from '../';
import mockData from '../../assets/mockData.json';

describe('ScheduleItemHeader', () => {
  it('renders correctly', () => {
    const comp = <ScheduleItemHeader author={mockData.author.name} title={mockData.title} />;

    const tree = renderer.create(comp).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
