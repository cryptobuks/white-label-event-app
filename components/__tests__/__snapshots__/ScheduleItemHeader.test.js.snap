// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScheduleItemHeader renders correctly 1`] = `
<View
  style={
    Object {
      "height": 40,
      "justifyContent": "space-between",
      "width": 200,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "fontSize": 16,
        "fontWeight": "600",
      }
    }
  >
    Cedric Devroey
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "fontSize": 14,
        "fontWeight": "600",
      }
    }
  >
    Bridging the Gap
  </Text>
</View>
`;
